scripts/data_oep_upload.py

```diff
@@ -0,0 +1,103 @@
+from oem2orm import oep_oedialect_oem2orm as oem2orm
+import os
+import pandas as pd
+
+
+### Setting up the oem2orm logger
+''' If you want to see detailed runtime information on oem2orm functions or if errors occur, you can activate the logger with this simple setup function. '''
+# oem2orm.setup_logger()
+
+# To connect to the OEP you need your OEP Token and user name. Note: You ca view your token on your OEP profile page after [logging in](https://openenergy-platform.org/user/login/?next=/). The following command will prompt you for your token and store it as an environment variable. When you paste it here, it will only show dots instead of the actual string.
+# save user name & token in environment as OEP_TOKEN & OEP_USER
+
+db = oem2orm.setup_db_connection()
+
+### Creating sql tables from oemetadata
+"""
+The oemetadata format is a standardised json file format and required for all data uploaded to the OEP. It includes the data model and the used data types. This allows us to derive the necessary tables in sqlalchemy from it.
+In order to create the table(s) we need to tell python where to find our oemetadata file first. To do this we place them in the folder "metadata" which is in the current directory (Path of this jupyter notebbok). Provide the path to your own folder if you want to use your own metadata. oem2orm will process all files that are located in the folder.
+"""
+
+metadata_folder = oem2orm.select_oem_dir(oem_folder_name="metadata")
+
+# The next command will set up the table. The collect_tables_function collects all metadata files in a folder and retrives the SQLAlchemy ORM objects and returns them. The Tables are ordered by foreign key. Having a valid metadata strings is necessary for the following steps.
+tables_orm = oem2orm.collect_tables_from_oem(db, metadata_folder)
+
+# create table
+oem2orm.create_tables(db, tables_orm)
+
+#%debug
+
+## Writing data into a table
+'''
+In this example we will upload data from a csv file. Pandas has a read_csv function which makes importing a csv-file rather comfortable. It reads csv into a DataFrame. By default, it assumes that the fields are comma-separated. Our example file has columns with semicolons as separators, so we have to specify this when reading the file.
+The example file for this tutorial ('upload_tutorial_example_data.csv') is in the 'data' directory, next to this tutorial. Make sure to adapt the path to the file you're using if your file is located elsewhere.
+'''
+
+filepath = "./metadata/"
+
+list_filenames = os.listdir(filepath)
+
+files = [filename.split(".")[0] for filename in list_filenames]
+
+print('This files will be uploaded \n')
```

Comment:
The scripts in oemof-B3 use logging, not print. Can we adapt this?

Reply:
Yes, from my side.